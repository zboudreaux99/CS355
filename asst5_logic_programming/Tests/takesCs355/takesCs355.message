Output line 0:
  <i>expected: </i> Miller, Crystal Mills, Jinghua Minor, Glenn Mitchell, Jeremiah <span class='expected'>Poe</span>, Angela Porter, Aretha Raines, Corrigan Reasor, Hamid Reyes, Gu
  <i>observed: </i> Miller, Crystal Mills, Jinghua Minor, Glenn Mitchell, Jeremiah <span class='observed'>no</span> Poe, Angela Porter, Aretha Raines, Corrigan Reasor, Hamid Reyes
on std err:
/home/zeil/bin/gprolog: 2: PATH: not found

