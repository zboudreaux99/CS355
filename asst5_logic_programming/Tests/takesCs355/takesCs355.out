uncaught exception: error(existence_error(procedure,takes/2),top_level/0)
